cmd/tink-worker/tink/tink_test.go

```diff
@@ -0,0 +1,264 @@
+package tink
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/pkg/errors"
+	pb "github.com/tinkerbell/tink/protos/workflow"
+	"google.golang.org/grpc"
+)
+
+type mockCli struct {
+	pb.WorkflowServiceClient
+	response     pb.WorkflowService_GetWorkflowContextsClient
+	actionstatus pb.WorkflowActionStatus
+}
+
+func (mcli *mockCli) GetWorkflowContexts(_ context.Context, _ *pb.WorkflowContextRequest, _ ...grpc.CallOption) (pb.WorkflowService_GetWorkflowContextsClient, error) {
+	return mcli.response, nil
+}
+
+func (mcli *mockCli) GetWorkflowActions(_ context.Context, _ *pb.WorkflowActionsRequest, _ ...grpc.CallOption) (*pb.WorkflowActionList, error) {
+	return nil, nil
```

Comment:
might think about testing a non-nil error return. as a side effect, you could get 100% code cover by doing so.